### DISCUSSION TITLE
Hyper.js configuration update alert keeps appearing

### DISCUSSION THREAD

**Discussion by dtun** [vercel/vercel#5147] [Help] [answered]
This alert appears every time I interact with the app.

<img width="380" alt="Screen Shot 2020-09-04 at 8 30 43 AM" src="https://user-images.githubusercontent.com/16211823/92256942-42fc9080-ee89-11ea-8c9a-edead563cc55.png">

When I say "every time I interact" I mean run a z command, open a tab- seems like every interaction triggers it.

Tried updating. Tried re-installing. Tried installing latest. Any other ideas on how to resolve it?

**Comment by nomad3k** [comment]
Oh, it's not just me.  This is driving me loopy! :)

**Comment by paulogdm** [comment, marked as answer]
Please create an issue in the Hyper.js repository: https://github.com/vercel/hyper

We are also looking for maintainers :smiley:
